src/select.coffee

```diff
@@ -213,6 +213,10 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   contentProxy: Ember.computed ->
     optionLabelPath = @get('optionLabelPath')
+    if optionLabelPath is ''
```

Comment:
When you're updating the `dist` files, you might also want to do `unless optionLabelPath` here to handle null/undefined. Sorry, should have caught that earlier.